### ISSUE TITLE
Masked Language modelling only predicts padding tokens

### ISSUE THREAD

**Issue by RAXAI** [CyberZHG/keras-bert#133]
Hi,

my main problem is my bert model only outputs padding tokens. since most of the input is padding tokens its learnt to only predict padding tokens therefore giving low loss. How do I stop the model from attending to the padding token. and focus on the real tokens. I tried to use your masking layer but do not not know how to use this . could you provide an example and solution? or tell me how i should modify my code.

also am I using the correct loss function for masked language modelling.

![bert_q](https://user-images.githubusercontent.com/32540275/64027786-5a4bc400-cb39-11e9-98f4-fd50f48629fd.png)
